Yoast/yoast-components
# Implement a loader using the Yoast logo

It is implemented using CSS animation by experimenting with the values.

How to test this;

* Click the Loader button in the App and see an animated Yoast logo that scales and fades in a rhythmic fashion.
